### PR TITLE
adding example of protocol_handlers

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

**Comment by tsl143** [comment]
Hi @rebloor , can you review this?

**Comment by rebloor** [comment]
@tsl143 I expect to do this next week.

File: open-irc-links/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# open-irc-links
+
+
+## What it does
+
+This add-on sets default client for opening IRC links using protocol_handlers.
```
Comment (rebloor) [inline open-irc-links/README.md:6]: Please add "the" before "default" i.e. "… sets the default client …"

File: open-irc-links/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Whenever any link with IRC protocol is clicked it opens with pre-defined URI template.
```
Comment (rebloor) [inline open-irc-links/README.md:8]: Please change to "Whenever a link using the IRC protocol is clicked, the link is opened in the URI defined in the add-on's manifest. In this example, all IRC protocol links are opened in mibbit.com."

File: open-irc-links/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+For example click [here](irc://irc.freenode.net/drupal).
```
Comment (rebloor) [inline open-irc-links/README.md:9]: Please change to "For example, click [this link to irc://irc.freenode.net/drupal](irc://irc.freenode.net/drupal) and the protocol handler opens the Drupal channel in mibbit.com."

File: open-irc-links/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## What it shows
+
+* usage of protocol_handlers in webextension.
```
Comment (rebloor) [inline open-irc-links/README.md:13]: Please change to "How to use protocol handlers to pass content to an application or website designed to handle that content."

File: open-irc-links/manifest.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "manifest_version": 2,
+  "name": "IRC Protocol Handler",
+  "description": "Demo the usage of protocol_handlers",
```
Comment (rebloor) [inline open-irc-links/manifest.json:4]: Please change to "Demos the use of protocol_handlers"

File: examples.json [hunk 6]
```diff
@@ -582,5 +582,10 @@
             "windows​.Window"
         ],
         "name": "window-manipulator"
+    },
+    {
+        "description": "Demonstrates usage of the protocol handlers.",
```
Comment (rebloor) [inline examples.json:587]: Please change to "Demonstrates the use of protocol handlers."

File: open-irc-links/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# open-irc-links
+
+
+## What it does
+
+This add-on sets the default client for opening IRC links using protocol_handlers.
```
Comment (rebloor) [inline open-irc-links/README.md:6]: Sorry, please format `protocol_handlers` as code.

File: open-irc-links/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+Whenever a link using the IRC protocol is clicked, the link is opened in the URI defined in the add-on's manifest.
+In this example, all IRC protocol links are opened in mibbit.com.  
+For example, click [here](irc://irc.freenode.net/drupal) and the protocol handler opens the Drupal channel in mibbit.com.
```
Comment (rebloor) [inline open-irc-links/README.md:10]: Please expose the URL so the reader can see what they clicking. Also, we should probably point out they need to install the extension first:
"For example, once you've installed the extension, open this link [irc://irc.freenode.net/drupal](irc://irc.freenode.net/drupal) and the protocol handler opens the Drupal channel in mibbit.com."

**Pull request merged**